File: CursoJava/src/Exer/EstruturaCondicional/Exer04.java
package Exer.EstruturaCondicional;

import java.util.Scanner;

public class Exer04 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.print("Digite a hora inicial do jogo (0 a 23): ");
		int horaInicial = sc.nextInt();

		System.out.print("Digite a hora final do jogo (0 a 23): ");
		int horaFinal = sc.nextInt();

		int duracao;

		if (horaInicial < horaFinal) {
			duracao = horaFinal - horaInicial;
		} else {
			duracao = 24 - horaInicial + horaFinal;
		}

		System.out.println("Duracao do jogo: " + duracao + " hora(s)");

		sc.close();
	}
}

File: CursoCsharpUdemy/exer/udemy/EstruturaCondicional/Exer04.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.udemy.EstruturaCondicional
{
    public class Exer04
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe a hora inicial do jogo:");
            int start = int.Parse(Console.ReadLine());

            Console.WriteLine("Informe a hora final do jogo:");
            int final = int.Parse(Console.ReadLine());

            int duration = 0;
            if (start < final)
            {
                duration = final - start;
            }
            else {
                duration = 24 - start + final;
            }

            Console.WriteLine($"O jogo durou {duration} hora(s)");
        }
    }
}